package.json

```diff
@@ -26,6 +26,7 @@
   "devDependencies": {
     "broccoli-asset-rev": "2.4.2",
     "csscomb": "3.1.8",
+    "cst": "0.3.0",
```

Comment:
@kevinansfield I didn't appear to need this locally, but without this addition it kept failing on Travis.